linalg/src/frame/by_scalar.rs

```diff
@@ -1,3 +1,61 @@
+use std::{fmt::Debug, marker::PhantomData};
+
+use tract_data::{TractResult, internal::TensorView};
+
+use crate::{LADatum, element_wise::ElementWiseKer, LinalgFn};
+
+use super::{ElementWise, element_wise_helper::map_slice_with_alignment};
+
+
+/// Generic implementation struct that unify all by scalar kernels.
+/// A by scalar operation is an ElementWise operation with a scalar paramerer.
+#[derive(Debug, Clone, new)]
+pub struct ByScalarImpl<K, T>
```

Comment:
I'm slowing moving away from this pattern in the matmul code. We may need to discuss it. (mmm kernels are now represented by the instance of a struct, lazy-initialized const)